src/main/java/com/example/movieapp/controllers/ProducerController.java

```diff
@@ -0,0 +1,28 @@
+package com.example.movieapp.controllers;
+
+import com.example.movieapp.entities.Producer;
+import com.example.movieapp.repositories.ProducerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/producers")
+public class ProducerController {
+
+    @Autowired
+    ProducerRepository producerRepository;
```

Comment:
you shouldn't address repo level directly from controller, the layering should be controller->service->repo